### ISSUE TITLE
Possibly add support for other RSS calendars?

### ISSUE THREAD

**Issue by michaelgira23** [michaelgira23/MyMICDS-v2#31]
We already have Portal and Canvas feeds, maybe extend it to any RSS feed too?

**Comment by michaelgira23** [comment]
A student just suggested this again. Might be higher on the priority list.

**Comment by nickbclifford** [comment]
iCal ≠ RSS, we might want to focus on iCal specifically since
- I believe most calendar apps can already export to iCal
- We already have the infrastructure mostly in place

**Comment by michaelgira23** [comment]
You're right, that's what I should've originally put. I can edit the issue now.
